tests/settings.py

```diff
@@ -82,18 +82,6 @@
     }
 ]
 
-
-# Using DatabaseCache to make sure that the cache is cleared between tests.
-# This prevents false-positives in some wagtail core tests where we are
-# changing the 'wagtail_root_paths' key which may cause future tests to fail.
-CACHES = {
-    "default": {
-        "BACKEND": "django.core.cache.backends.db.DatabaseCache",
-        "LOCATION": "cache",
-    }
-}
-
-
```

Comment:
@katdom13 thanks a lot for your help! :+1: 

I'm not completely sure, if calling runserver that way on the testapp is supported. This setup is mainly based on https://github.com/wagtail/cookiecutter-wagtail-package and to run a testserver you normally use `tox -e interactive` (I should have placed this in the development section of the README :sweat_smile: )
That said, I'm also not sure if removing the db-cache has some implications on future tests :thinking:

Reply:
I looked into this.

Because the cache it set to: `django.core.cache.backends.db.DatabaseCache` there's a command that needs to be run to create the cache table `python manage.py createcachetable` which would solve the error.

I also noticed that DEBUG is set via an env var `INTERACTIVE` which isn't set so DEBUG is False therefore the static assets don't load so running the testapp with `python manage.py runserver' doesn't work as it should. It is set when running the tox testing though.

Some options to use a none DB cache would be: (un-tested)

```python

CACHES={
    "default": {
        "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
    }
}
    
or

CACHES={"default": {"BACKEND": "django.core.cache.backends.dummy.DummyCache"}}

```

What I would do here is not make this change add some documentation to implement a `local.py` settings file that's imported into settings like so.

```python
# add this to the current settings file at the end
try:
    from .local import *  # NOQA
except ImportError:
    pass
```
add a local.example.py settings file and add
```
DEBUG=True
```
Then describe in some setup docs to copy `local.example.py` to `local.py`

This way you can then use the `runserver` command to use the testapp

Reply:
Hi @nickmoreton , @th3hamm0r ,
I pushed a new commit to resolve this.
I left the cache settings as is previously and spun up a `local.py.example` to override it.
I also added some docs regarding on how to run the local app and to copy the new `local.py.example`.

Please let me know if I forgot something.

Thank you for reviewing!